projects/swimlane/ngx-graph/src/lib/graph/graph.component.ts

```diff
@@ -94,6 +95,10 @@ export class GraphComponent extends BaseChartComponent implements OnInit, OnChan
   @Input() layout: string | Layout;
   @Input() layoutSettings: any;
   @Input() enableTrackpadSupport = false;
+  @Input() keyboardPanValue: number = 50;
+  @Input() zoomKeyboardIncrease: number = 1.2;
+  @Input() zoomKeyboardDecrease: number = 0.8;
```

Comment:
Any reason we want different inputs for these?

Reply:
I assume it is fine tuning on the zoom speed on both ways, zoom in and out, so users can have control of those separately.